src/core/cardano/walletConnect/identityWalletConnect.ts

```diff
@@ -93,7 +123,16 @@ class IdentityWalletConnect extends CardanoPeerConnect {
     addr: string,
     payload: string
   ): Promise<Cip30DataSignature> {
-    throw new Error("Method not implemented.");
+    const toHex = (text: string) =>
+      text
+        .split("")
+        .map((char) => char.charCodeAt(0).toString(16))
+        .join("");
+    const signature = await this.sign(addr, payload);
+    return {
+      key: toHex(addr),
+      signature,
+    };
```

Comment:
AFAIK we don't need to implement this. This will be used by normal crypto wallets, and our `.sign` method will be used for identity related things.

Reply:
This has been reverted